### DISCUSSION TITLE
[@types/react] Doesn't allow `autoFocus` on any element

### DISCUSSION THREAD

**Discussion by TomerAberbach** [DefinitelyTyped/DefinitelyTyped#64690] [Issues with a @types package]
[`autoFocus`](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/autofocus) is a "global attribute" and is useful for any element that's focusable, but right now `@types/react` doesn't include it on many `JSX.IntrinsicElements` types. I noticed this while trying to add `autoFocus` to an `a` element.

[a comment by an automated account is not shown]

**Comment by eps1lon** [comment]
Fixed once https://github.com/DefinitelyTyped/DefinitelyTyped/pull/64881 lands

**Reply by TomerAberbach** [reply]
Thanks!
